src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/RazorSemanticTokenInfoService.cs

```diff
@@ -9,10 +9,6 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic
 {
     internal abstract class RazorSemanticTokensInfoService
     {
-        public abstract Task<SemanticTokens?> GetSemanticTokensAsync(TextDocumentIdentifier textDocumentIdentifier, CancellationToken cancellationToken);
-
-        public abstract Task<SemanticTokens?> GetSemanticTokensAsync(TextDocumentIdentifier textDocumentIdentifier, Range? range, CancellationToken cancellationToken);
-
-        public abstract Task<SemanticTokensFullOrDelta?> GetSemanticTokensEditsAsync(TextDocumentIdentifier textDocumentIdentifier, string? previousId, CancellationToken cancellationToken);
+        public abstract Task<SemanticTokens?> GetSemanticTokensRangeAsync(TextDocumentIdentifier textDocumentIdentifier, Range range, CancellationToken cancellationToken);
```

Comment:
Given there's only one method now we could probably just assume range is default and re-purpose `GetSemanticTokensAsync`